simplivity/resources/hosts.py

```diff
@@ -180,3 +180,22 @@ def get_virtual_controller_shutdown_status(self):
         """Retrieves the shutdown status of the Virtual Controller"""
         resource_uri = "{}/{}/virtual_controller_shutdown_status".format(URL, self.data["id"])
         return self._client.do_get(resource_uri)
+
+    def shutdown_virtual_controller(self, ha_wait=True, timeout=-1):
+        """Shuts down the Virtual Controller safely (by reaching HA compliance) or by force.
+
+        Args:
+          ha_wait: An indicator to show if the user wants to shut down the Virtual Controller safely or forcefully.
+            Valid values:
+              True: Virtual Controller waits for the virtual machines to reach HA compliance before shutting down.
+              False: Virtual Controller forced to shut down without waiting for HA compliance.
+          timeout: Time out for the request in seconds.
+
+        Returns:
+            object: status object.
```

Comment:
This isn't an object, it is just a python `dict`

Reply:
Please check the feedback on the cancel shutdown as some of them apply here.

Reply:
Should we just make this a string

Reply:
Updated to code suggested